conftest.py

```diff
@@ -31,6 +31,16 @@ def pytest_sessionstart():
     downloader.main(['-d', 'sct_testing_data', '-o', sct_test_path()])
 
 
+@pytest.fixture
+def run_in_sct_testing_data_dir():
+    """Temporarily change the working directory to 'sct_testing_data'. This replicates the behavior of the old
+    `sct_testing`, and is needed to prevent tests from cluttering the working directory with output files."""
+    cwd = os.getcwd()
+    os.chdir(sct_test_path())
+    yield
+    os.chdir(cwd)
```

Comment:
I think you can move this to `unit_testing/cli/conftest.py`, **and mark it with** `autouse=True`. Then you can get rid of all the `@pytest.mark.usefixtures()`.

`conftests` are all run according to the folder structure, the way packages all run their `__init__`s: https://docs.pytest.org/en/latest/how-to/fixtures.html?highlight=conftest#override-a-fixture-on-a-folder-conftest-level

Reply:
Though _explicit is better than implicit_

Reply:
The tricky thing with this is that it would apply to all of the existing pytest tests (i.e. the non-sct_testing ones).

I don't think we handle `sct_test_data()` / `sct_testing_data` consistently in the test suite. So, I think it would be easiest to handle this in #3422.

Reply:
I think if you specifically put it under `unit_testing/cli`, then it only applies to `unit_testing/cli`. If there's non-sct_testing tests in there already (I haven't looked to see) then you could make a separate folder just for them.